BlazarService/src/main/java/com/hubspot/blazar/util/GitHubWebhookHandler.java

```diff
@@ -106,11 +108,7 @@ public void handlePushEvent(PushEvent pushEvent) throws IOException {
 
     if (!pushEvent.getRef().startsWith("refs/tags/") && !pushEvent.getDeleted()) {
```

Comment:
@jhaber The check for `.getDeleted()` here prevents us from re-activating branches that were deleted.